Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -239,6 +239,14 @@ bool LogManager::IsEnabled(LOG_TYPE type, LOG_LEVELS level) const
   return m_log[type].m_enable && GetLogLevel() >= level;
 }
 
+void LogManager::GetLogTypes(std::map<std::string, std::string>& logTypes)
```

Comment:
Consider making this return a map instead of appending to one that was passed in as an parameter. If not, you should rename the parameter to `log_types` and make it use a pointer instead of a reference, both per Contributing.md.